### ISSUE TITLE
Use SBML to LaTeX converter which supports qualitative models

### ISSUE THREAD

**Issue by eikept** [draeger-lab/SBMLsqueezer#15]
Currently only quantitative models are supported for printing reports as a LaTeX document. [SBMLreporter](https://github.com/sbmlteam/SBMLreporter) could be a possible option for implementing such a feature (and therefore replace SBML2LaTeX) in the future although currently it's development is not sufficiently advanced.

**Comment by draeger** [comment]
Indeed. SBMLreqporter is a new development and designed to replace SBML2LaTeX. At the moment, however, SBMLreqporter needs further development and does not seem to be ready yet.
